src/main/java/org/owasp/esapi/reference/validation/HTMLValidationRule.java

```diff
@@ -46,22 +46,29 @@
 	/** OWASP AntiSamy markup verification policy */
 	private static Policy antiSamyPolicy = null;
 	private static final Logger LOGGER = ESAPI.getLogger( "HTMLValidationRule" );
+	private static final String ANTISAMYPOLICY_FILENAME = "antisamy-esapi.xml";
 
 	static {
         InputStream resourceStream = null;
 		try {
-			resourceStream = ESAPI.securityConfiguration().getResourceStream("antisamy-esapi.xml");
+			resourceStream = ESAPI.securityConfiguration().getResourceStream(ANTISAMYPOLICY_FILENAME);
```

Comment:
I dunno. Seems to me if you are going to go through all this trouble to load this via the class path, that you might also want to consider a way of specifying an alternate filename. Because everyone may not want to call there's "antisamy-esapi.xml". Because as I mentioned, what if you find multiple, but different files with that name in the class path.

Call me ancient, but all this reminds me of "Colossal Cave Adventure": "You're in a maze of twisty little passages, all alike." :)

Reply:
I was only solving issue 310 and not enhancing the code ;-)

Reply:
Fair enough, but whether it was your intent or not, you did "enhance" the code by adding a new method to the SecurityConfiguration interface. I likely would have commented otherwise had you only used a private method from HTMLValidationRule instead.

Reply:
Resolved (sort off; will add some new general comments later) by providing a new ESAPI property named "Validator.HtmlValidationConfigurationFile". (There is still a problem that this new property is not documented in either of the ESAPI.properties files, but that is what I will comment on elsewhere.)